docs/source/geopm_agent_ffnet.7.rst

```diff
@@ -16,27 +15,13 @@ per-domain frequency. The user can specify an energy-performance bias in the
 agent policy, which determines the degree to which the frequency recommender is
 willing to potentially reducing performance by reducing frequency to save energy.
 
-This agent requires a neural net JSON file and a region hint recommender JSON file.
-Without these JSON files, the agent will throw an error. These files can be autogenerated
-using scripts included in the ``$GEOPM_SOURCE/integration/experiment/ffnet`` 
-directory, as described below.
-
-The neural net for region classification must be provided in a JSON configuration
-file pointed to by environment variables ``GEOPM_CPU_NN_PATH`` and/or 
-``GEOPM_GPU_NN_PATH``, which must comply with the following schema:
-
-.. literalinclude:: ../json_schemas/domainnetmap_neural_net.schema.json
-    :language: json
-
-The per-region frequency recommendations must be provided in a JSON configuration
-file pointed to by environment variables ``GEOPM_CPU_FMAP_PATH`` and/or 
-``GEOPM_GPU_FMAP_PATH``, which must comply with the following schema:
+This agent requires a directory specified by the environment variable ``GEOPM_FFNET_PATH``.
+The directory must contain the following files:
 
-.. literalinclude:: ../json_schemas/regionhintrecommender_fmap.schema.json
-    :language: json
+- Neural net JSON files with the suffix ``*_nn_cpu.json`` and/or ``*_nn_gpu.json``.
+- Frequency recommendation map JSON files with the suffix ``*_fmap_cpu.json`` and/or ``*_fmap_gpu.json``.
 
-If you specify a neural net file for a domain (CPU/GPU), you must specify a frequency 
-recommendation file as well.
+If you specify a neural net file for a domain (CPU/GPU), you must also provide a corresponding frequency recommendation file.
```

Comment:
Might need to add policy parameters, e.g. FFNET_PACKAGE_ENABLE / FFNET_GPU_ENABLE. Otherwise user would need to delete or move files depending on the domains they wish to characterize/steer.